packages/nextjs-template/components/DendronLayout.tsx

```diff
@@ -103,6 +103,7 @@ export default function DendronLayout(
             // initialize
             mermaid.init();
           }}
+          strategy="lazyOnload"
```

Comment:
Not very significant, but it would load script during browser idle time. With the default strategy also, the script is loaded after the page becomes interactive.

Reply:
how does this affect existing pages with mermaid near the top of the page? does it cause a pop in effect?

Reply:
Yeah, similar to the way it shows up now. https://docs.dendron.so/notes/446723ba-c310-4302-a651-df14ce6e002b/